README.org

```diff
@@ -20,7 +20,7 @@ For the documentation, see https://input-output-hk.github.io/haskell.nix/.
 For =cabal.project= project add a =default.nix=:
 
 #+begin_src sh
-{ pkgs ? import <nixpkgs> (import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz))
+{ pkgs ? import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz + "/nixpkgs") {}
```

Comment:
I think this kind of store-path construction can get you into trouble on hydra when evaluating in restricted mode. I've seen it get unhappy about you accessing `<derivation> + <path>` before.

Can we not just provide the source derivation as an *attribute*?

Reply:
Oh rats, our `default.nix` just *is* the nixpkgs arguments, there's no space to have extra attributes :/

I suppose it's too late to refactor `default.nix` to be `{ nixpkgsArgs = ...; nixpkgs-pins = ... }`?

Reply:
Yeah I was just thinking about that too.

Reply:
Even if we don't *need* it now, it's nice to have the flexibility to be able to include "other useful stuff" in `haskell.nix` easily. Obviously most of that can go in the overlay, but for example we could expose a set of tested `hackage.nix` sources or something too.